basics/transfer-sol/seahorse/programs_py/seahorse.py

```diff
@@ -0,0 +1,26 @@
+# seahorse
+# Built with Seahorse v0.2.7
+
+from seahorse.prelude import *
+
+declare_id('2RjL4mpTANyGxz7fLWEbQtmdEDti7c4CqsLR96mgvcaV')
+
+class MockAccount(Account):
+    mock_value: u8
+
+
+## We don't care about this instruction, but we need an address to send our SOL to.
+@instruction
+def init_mock_account(signer: Signer, mock_account: Empty[MockAccount]):
+    account = mock_account.init(
+        payer = signer,
+        seeds = ['mock_account'],
+    )
+    account.mock_value = u8(0)
+
+@instruction
+def transfer(sender: Signer, recipient: MockAccount, amount: u64):
```

Comment:
Can we use the same pattern & naming convention as the other two transfer examples? 

There's both a "transfer with CPI" and "transfer with program" instruction in each, and in both cases the account names are `payer` and `recipient`